src/main/scala/codacy/scalastyle/ScalaStyle.scala

```diff
@@ -274,10 +275,16 @@ object ScalaStyle extends Tool {
     FileHelper.createTmpFile(scalaStyleNewConfig).toFile
   }
 
+  private def jsValueToString(value: JsValue) = {
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [jsValueToString should have a smaller scope by using private[this]](https://www.codacy.com/app/Codacy/codacy-scalastyle/file/2721206641/issues/source?bid=3135657&fileBranchId=3347714#l278)